lib/services/db.js

```diff
@@ -215,6 +216,91 @@ function formatBatchOperations(ops) {
   }).join('\n');
 }
 
+/**
+ *
+ * @param {object} hookOps
+ * @param {Array} hookOps.put
+ * @param {Array} hookOps.del
+ * @param {object} op
+ * @returns {object}
+ */
+function addToHookOps(hookOps, op) {
+  if (hookOps[op.type]) {
+    hookOps[op.type].push(_.pick(op, 'type', 'key', 'value'));
```

Comment:
Two questions?

1. Any reason to not just `return hookOps.concat(_.pick(op, 'type', 'key', 'value'))`
2. Why use `_.pick` here? The API for the an op is consistent, unless there's something I'm not thinking of.